### DISCUSSION TITLE
Renovate config for OpenTofu

### DISCUSSION THREAD

**Discussion by kimdre** [opentofu/opentofu#1761] [Q&A]
Hi,
the Renovate bot tries to update the
```yaml
required_version = ">= 1.7.2"
```
section in my repos all the time to the latest terraform version (1.9.0).

Does someone know how I can change this behaviour?

**Comment by ghost** [comment]
It seems that Renovate [already supports OpenTofu](https://github.com/renovatebot/renovate/blob/622c604610d69230efa592a92d91cd9caa282863/lib/modules/manager/terraform/readme.md), but [looking at the code](https://github.com/renovatebot/renovate/blob/622c604610d69230efa592a92d91cd9caa282863/lib/modules/manager/terraform/extractors/terraform-block/terraform-version.ts#L38) it seems that they only fetch releases from the Terraform repository and I haven't found an override for this. It may be worth opening an issue with them.

It is also worth noting that they should probably add support for the [OpenTofu-specific code overrides](https://github.com/opentofu/opentofu/blob/main/rfc/20240529-OpenTofu-Specific-Code-Override.md) coming in 1.8.

**Comment by kimdre** [comment]
There is a discussion in the Renovate Repository regarding this: https://github.com/renovatebot/renovate/discussions/29786
And also a workaround/"temporary" solution that replaces the `depName` with opentofu: https://github.com/renovatebot/renovate/discussions/29786#discussioncomment-9836492
